src/components/Person/Person.jsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 
-export const Person = ({ person }) => (
-  <section className="Person">
-    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
-    {person.age ? <p className="Person__age">{`I am ${person.age}`}</p> : ''}
-    <p className="Person__partner">
-      {/* eslint-disable-next-line no-nested-ternary */}
-      {person.isMarried
-        ? person.sex === 'm'
-          ? `${person.partnerName} is my wife`
-          : `${person.partnerName} is my husband`
-        : 'I am not married'}
-    </p>
-  </section>
-);
+export const Person = ({ person }) => {
+  const { name, age, isMarried, partnerName, sex } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
+          : 'I am not married'}
```

Comment:
it is still nested ternary operator)

let's avoid it
let's create a variable
```javascript
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
``` 
and then use it

```javascript
{isMarried
          ? `${partnerName} is my ${personPartnerType}`
          : 'I am not married'}
```